src/commands/gist.ts

```diff
@@ -402,6 +402,43 @@ export async function registerGistCommands(context: ExtensionContext) {
     )
   );
 
+  context.subscriptions.push(
+    commands.registerCommand(
+      `${EXTENSION_NAME}.viewForkedGist`,
+      async (node?: StarredGistNode | FollowedUserGistNode) => {
+        await ensureAuthenticated();
```

Comment:
Does the API require you to be authenticated in order to view forks? If not, we could potentially remove this.

Reply:
Good point. I'll double check.